py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -169,55 +169,182 @@ def aten_ops_gelu(
     )
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.matmul)
-@dynamo_tensorrt_converter(torch.ops.aten.mm.default)
-def aten_ops_matmul(
+@dynamo_tensorrt_converter(torch.ops.aten.relu.default)
+def aten_ops_relu(
     network: TRTNetwork,
     target: Target,
     args: Tuple[Argument, ...],
     kwargs: Dict[str, Argument],
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
-    return impl.matmul.matrix_multiply(
-        network, target, SourceIR.ATEN, name, args[0], args[1]
+    return impl.actv.relu(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
     )
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.layer_norm.default)
-def aten_ops_layernorm(
+@dynamo_tensorrt_converter(torch.ops.aten.sigmoid.default)
+def aten_ops_sigmoid(
     network: TRTNetwork,
     target: Target,
     args: Tuple[Argument, ...],
     kwargs: Dict[str, Argument],
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
-    return impl.normalization.layer_norm(
+    return impl.actv.sigmoid(
```

Comment:
use the full name (activation)

Reply:
Done, but got error:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/__init__.py", line 86, in <module>
    from torch_tensorrt._compile import *  # noqa: F403
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/_compile.py", line 13, in <module>
    from torch_tensorrt.dynamo.compile import compile as dynamo_compile
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/__init__.py", line 13, in <module>
    from .compile import compile  # noqa: F403
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/compile.py", line 13, in <module>
    from torch_tensorrt.dynamo import CompilationSettings, partitioning
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/partitioning/__init__.py", line 1, in <module>
    from ._adjacency_partitioner import partition as fast_partition
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/partitioning/_adjacency_partitioner.py", line 16, in <module>
    from torch_tensorrt.dynamo.conversion.converter_registry import (
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/conversion/__init__.py", line 2, in <module>
    from .aten_ops_converters import *  # noqa: F403
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/conversion/aten_ops_converters.py", line 8, in <module>
    from torch_tensorrt.dynamo.conversion import impl
  File "/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/conversion/impl/__init__.py", line 3, in <module>
    from . import (
ImportError: cannot import name 'activation' from partially initialized module 'torch_tensorrt.dynamo.conversion.impl' (most likely due to a circular import) (/opt/circleci/.pyenv/versions/3.10.9/lib/python3.10/site-packages/torch_tensorrt/dynamo/conversion/impl/__init__.py)

Exited with code exit status 1
CircleCI received exit code 1
```

Did you come across the similar error by any chance?

Reply:
I've seen this error before. The issue seems to be here:
https://github.com/pytorch/TensorRT/blob/cd7e98d9e23843138e1dc887f52a744995fec197/py/torch_tensorrt/dynamo/conversion/impl/activation/ops.py#L10
It is trying to import a function which is being initialized. I think it can be fixed by replacing with
```python
from .base import convert_activation
```

Reply:
Thanks George! I modified as you suggested but it looks like not the problem. I got the same error.

Reply:
Thanks for testing this - I actually tried this branch on my own machine and I'm not seeing this error, with either the `from torch_tensorrt.dynamo.conversion.impl.activation.base import convert_activation` or `from .base import convert_activation`. Do you see the error locally?

Reply:
Thanks for your help! No, I don't. I typically push every time after passing all tests on my local machine. That's weird...

Reply:
No problem! Definitely strange - could try a rebase to `main` to resolve the merge conflict and see if anything changes with that.

Reply:
I rebased, but it doesn't work (I used `from .base import convert_activation`). I guess it's not the problem of `torch_tensorrt.dynamo.conversion.impl.activation.base import convert_activation` because the unary folder uses the similar `from torch_tensorrt.dynamo.conversion.impl.unary.base import convert_unary` but it works. :dizzy_face: